server/testutils/factories/placementRequest.ts

```diff
@@ -20,7 +20,7 @@ export default Factory.define<PlacementRequest>(() => ({
   risks: risksFactory.build(),
   applicationId: faker.datatype.uuid(),
   assessmentId: faker.datatype.uuid(),
-  releaseType: faker.helpers.arrayElement(['licence', 'rotl', 'hdc', 'pss', 'in_community']),
+  releaseType: faker.helpers.arrayElement(['licence', 'rotl', 'hdc', 'pss']),
```

Comment:
It would be clearer if this was typed correctly in the spec

Reply:
👍 Will update the API

Reply:
Actually, we DO need `in_community` here, so have tweaked slightly